tests/wp-unit/include/Core/Metas/Modules/DeleteUserMetaModuleTest.php

```diff
@@ -49,23 +54,30 @@ public function test_deleting_single_user_meta_fields_from_admin_user_role() {
 
 		$user_meta = get_user_meta( $user, 'time', true );
 		$this->assertEquals( '', $user_meta );
+
```

Comment:
We are deleting user meta and not users. So you should add another user meta for the same user and make sure that the additional meta is not deleted.

Do this for all the test cases.

Reply:
Fixed